packages/terra-table/src/terra-dev-site/doc/example/SectionExample.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Table from 'terra-table';
+
+const TableSectionExample = () => (
+  <Table
+    summaryId="section-table"
+    summary="This table displays section grouping."
+    paddingStyle="standard"
+    sectionData={[
+      {
+        sectionHeader: {
+          title: 'Default Section',
+          id: 'default-id',
+          key: 'default',
+          numberOfColumns: 3,
+        },
+        rows: [
+          {
+            key: 'row-0',
+            cells: [
+              {
+                key: 'cell-0',
+                children: [
+                  <div key="cell-0">Lorem ipsum dolor</div>,
+                ],
+              },
+              {
+                key: 'cell-1',
+                children: [
+                  <div key="cell-1">Lorem ipsum dolor</div>,
+                ],
+              },
+              {
+                key: 'cell-2',
+                children: [
+                  <div key="child">Lorem ipsum dolor</div>,
+                ],
+              },
+            ],
+          },
+        ],
+      },
+      {
+        sectionHeader: {
+          title: 'Collapsible Section',
+          id: 'collapsible-id',
+          key: 'collapsible',
+          isCollapsible: true,
+          numberOfColumns: 3,
+        },
+        rows: [
+          {
+            key: 'row-0',
+            cells: [
+              {
+                key: 'cell-0',
+                children: [
+                  <div key="cell-0">Lorem ipsum dolor</div>,
+                ],
+              },
+              {
+                key: 'cell-1',
+                children: [
+                  <div key="cell-1">Lorem ipsum dolor</div>,
+                ],
+              },
+              {
+                key: 'cell-2',
+                children: [
+                  <div key="child">Lorem ipsum dolor</div>,
+                ],
+              },
+            ],
+          },
+        ],
+      },
+      {
+        sectionHeader: {
```

Comment:
Maybe we need another example where the section header is _actually_ collapsible - having both `isCollapsed` and `isCollapsible` set to true? I do not see a difference in behaviour amongst the current examples. 

<img width="785" alt="Screen Shot 2019-12-10 at 4 51 17 PM" src="https://user-images.githubusercontent.com/10090859/70525572-b036fb80-1b6d-11ea-851b-333672380689.png">

Reply:
Updated.